tests/testthat/tests_calculations.R

```diff
@@ -0,0 +1,40 @@
+context("EGRET calculation functions")
+
+test_that("calculateMonthly", {
```

Comment:
ideally the first argument to each call to `test_that` would be an assertion, e.g., "calculateMonthly returns data of expected format". see http://r-pkgs.had.co.nz/tests.html#test-structure. i realize this will be a pain, but now is the best time to do it - while these new tests are fresh in your mind - and it'll help summarize which behaviors of each function you did or didn't test